tips.json

```diff
@@ -30,5 +30,23 @@
     "tool": "OS X",
     "toolSlug": "osx",
     "text": "[cmd + tab] will help you qickly cycle through your open applications instead of having to click on them from the dock. So fast!"
+  },
+  {
+    "tool": "EditorConfig",
+    "toolSlug": "editors",
+    "source": "http://editorconfig.org/",
+    "text": "You can use the .editorconfig format to maintain consistent coding styles between different editors and IDEs. Excellent if you are working on a group!"
```

Comment:
"working on a group" isn't wrong, but generally it's more common to hear "working in a group"

Reply:
Can you wrap the `.editorconfig` text in hard brackets? Like so:

"You can use the [.editorconfig] format to..."  This will give it a "code" syntax style when it's displayed in the extension and help it stand out a little more.